printers.tex

```diff
@@ -22,19 +24,21 @@
 
 \begin{verbatim}
 >>> pprint(Integral(sqrt(phi0 + 1), phi0), use_unicode=False)
-  /
- |
- |   __________
+  /                    
+ |                     
+ |   __________        
```

Comment:
My editor is going to clear these the next time I edit the file. Can we make the doctester ignore trailing whitespace?